plugins/advisors/black-duck/src/main/kotlin/BlackDuckOriginId.kt

```diff
@@ -58,3 +58,8 @@ internal data class BlackDuckOriginId(
         }
     }
 }
+
+// A replacement for `Forge.getKnownForges()`, because the latter does not contain entries for all forges.
+private val KNOWN_FORGES = Forge.getKnownForges() + listOf(
```

Comment:
Alternatively:

     private val KNOWN_FORGES = Forge.getKnownForges() + Forge(":", "conan").let { it.name to it }

Reply:
I'll need to add another forge to that list (in a following PR). So, I'd prefer to keep this a list for easy extension.